starter_kits/TypeScript/hlt/MapCell.ts

```diff
@@ -0,0 +1,54 @@
+import { Ship } from "./Ship";
+import { Position } from "./Position";
+import { Shipyard } from "./Shipyard";
+import { Dropoff } from "./Dropoff";
+
+/** A cell on the game map. */
+export class MapCell {
+    ship: Ship | undefined = undefined;
+    structure: Shipyard | Dropoff | undefined = undefined;
+    constructor(public position: Position, public haliteAmount: number) {
```

Comment:
Empty constructor, delete it?

Reply:
With the `public` modifiers before the arguments this expression defines and sets them as members.

Reply:
Didn't know that. Awesome.